### PR TITLE
Change cache type

### PR DESCRIPTION
**Pull request by yiiliveext**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ❌
| Breaks BC?    | ❌
| Tests pass?   | ✔️

### CODE REVIEW

File: config/common.php [hunk 1]
```diff
@@ -17,8 +23,10 @@
 $timer->start('overall');
 
 return [
-    \Psr\SimpleCache\CacheInterface::class => \Yiisoft\Cache\ArrayCache::class,
-    \Yiisoft\Cache\CacheInterface::class => \Yiisoft\Cache\Cache::class,
+    CacheInterface::class => function (ContainerInterface $container) {
+        return new FileCache($container->get(Aliases::class)->get('@runtime/cache'));
+    },
+    YiiCacheInterface::class => Cache::class,
```
Comment (roxblnfk) [inline config/common.php:29]: I phink full class name better than alias in this case

File: config/common.php [hunk 2]
```diff
@@ -17,8 +23,10 @@
 $timer->start('overall');
 
 return [
-    \Psr\SimpleCache\CacheInterface::class => \Yiisoft\Cache\ArrayCache::class,
-    \Yiisoft\Cache\CacheInterface::class => \Yiisoft\Cache\Cache::class,
+    CacheInterface::class => function (ContainerInterface $container) {
```
Comment (roxblnfk) [inline config/common.php:26]:
```suggestion
    CacheInterface::class => static function (ContainerInterface $container) {
```

**Comment by samdark** [comment]
👍

File: config/common.php (hunk 2 above)
Comment (samdark) [reply config/common.php:26]: c565c6b71f2c5a2797846b52f992dc69bd346d6f

**Pull request merged**